addons/mail/models/models.py

```diff
@@ -173,14 +174,20 @@ def _mail_track_order_fields(self, tracked_fields):
         fields_track_info.sort(key=lambda item: (item[1], item[0]), reverse=True)
         return fields_track_info
 
-    def _mail_track_get_field_sequence(self, fname):
+    def _mail_track_get_field_sequence(self, field):
```

Comment:
Why changing the API ? You can just check if fname is in `self._fields` and return the default if not.

Reply:
thank you bro. I have updated it. Please review again.